hack/olm-registry/olm-artifacts-template.fedramp.yaml

```diff
@@ -116,6 +116,14 @@ objects:
             - get
             - list
             - watch
+          - apiGroups:
+            - ""
+            resources:
+            - serviceaccounts/token
+            verbs:
+            - create
+            - update
+            - patch
```

Comment:
Just wanted to confirm if we need the update and patch verbs as well?

Reply:
No, I'll update this

Reply:
Removed unnecessary verbs